### PR TITLE
[IMP] sale: add setting section to display Shopee connector

### PR DESCRIPTION
**Pull request by stta-odoo**
* Add setting section for Shopee

task-3623720
Enterprise PR: https://github.com/odoo/enterprise/pull/71610

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale/wizard/res_config_settings.py [hunk 1]
```diff
@@ -72,6 +72,8 @@ class ResConfigSettings(models.TransientModel):
 
     module_product_email_template = fields.Boolean("Specific Email")
     module_sale_amazon = fields.Boolean("Amazon Sync")
+    module_sale_shopee = fields.Boolean("Shopee Sync")
+    module_sale_lazada = fields.Boolean("Lazada Sync")
```
Comment (ltinel) [inline addons/sale/wizard/res_config_settings.py:76]: I don't see any reference to Lazada in the enterprise PR. Is there a separate PR to handle it? If so I would split this PR (and update the PR/commit title and description).

**Review by ltinel — CHANGES_REQUESTED** [review]
Thanks for your work! I'll have a look at the enterprise PR next (it will take a bit of time though :sweat_smile:). Could you reference the enterprise PR in the description?

File: addons/sale/wizard/res_config_settings.py (hunk 1 above)
Comment (stta-odoo) [reply addons/sale/wizard/res_config_settings.py]: I've updated this PR and removed Lazada related code.

**Review by ltinel — APPROVED** [review]
LGTM :)

File: addons/sale/wizard/res_config_settings.py [hunk 2]
```diff
@@ -72,6 +72,7 @@ class ResConfigSettings(models.TransientModel):
 
     module_product_email_template = fields.Boolean("Specific Email")
     module_sale_amazon = fields.Boolean("Amazon Sync")
+    module_sale_shopee = fields.Boolean("Shopee Sync")
```
Comment (vava-odoo) [inline addons/sale/wizard/res_config_settings.py:75]: I don't think you can add this field in stable

**Pull request closed without merging**